packages/api/src/webhooks/cannon.ts

```diff
@@ -290,43 +375,102 @@ export default class WebhookCannon {
       // sign payload if there is a webhook secret
       if (webhook.sharedSecret) {
         let signature = sign(params.body, webhook.sharedSecret);
-        params.headers["Livepeer-Signature"] = `t=${timestamp},v1=${signature}`;
+        params.headers[SIGNATURE_HEADER] = `t=${timestamp},v1=${signature}`;
+      }
+      try {
+        await this.storeTriggerTime(webhook);
+      } catch (e) {
+        console.log(
+          `Unable to store trigger time of webhook ${webhook.id} url: ${webhook.url}`
+        );
       }
 
+      let resp: Response;
+      let errorMessage;
+      let statusCode;
```

Comment:
**nit:** Should add types to these
```
      let errorMessage: string;
      let statusCode: number;
```

Edit: Even better, instead of declaring individual fields, WDYT of declaring the "last failure" field itself?
```suggestion
      let lastFailure: Webhook["status"]["lastFailure"] = { timestamp: Date.now() };
```
(yes, indexing types works!)

Then when using it you could just set the fields on the object like `lastFailure.statusCode = resp.status` etc